### PR TITLE
Implement Passenger57 and Game class for the Roulette game.

### PR DESCRIPTION
**Pull request by FarzanRashid**
This PR includes Passenger57 and Game class for Roulette game. This PR also includes tests for Game class.

### CODE REVIEW

File: src/roulette.py [hunk 1]
```diff
@@ -674,3 +674,121 @@ def __repr__(self) -> str:
...
+           For each :class:`Bet` instance, if the winning :class:`Bin` contains the
+           :class:`Outcome`, call **Player.win()** method, otherwise, call the
+           **Player.lose()** method.
+        """
+
+        player.placeBets()
+        self.table.isValid()  # Ensures bets are valid.
+        winning_bin = self.wheel.choose()
+        bet_iterator = iter(self.table)
+        for bet in bet_iterator:
```
Comment (FarsanRashid) [inline src/roulette.py:790]: better directly iterate over `self.table`

File: tests/test_game.py
Comment (FarsanRashid) [inline tests/test_game.py:1]: a test can be added to check wheel will not spin when invalid bet is placed.

File: src/roulette.py (hunk 1 above)
Comment (FarzanRashid) [reply src/roulette.py]: Done.

File: tests/test_game.py
Comment (FarzanRashid) [reply tests/test_game.py:1]: Done.

**Pull request merged**